src/plugins/wizard/public/application/components/data_source_select.scss

```diff
@@ -0,0 +1,5 @@
+@import "../variables";
+
+.wizDatasourceSelect {
+   max-width: 469px;
```

Comment:
You've imported variables but have this as a fixed number? Also could this be why you need the negative padding to be different from the dropdown padding?